src/utils/merge.ts

```diff
@@ -0,0 +1,115 @@
+import mergeWith from 'lodash.mergewith';
+import cloneDeep from 'lodash.clonedeep';
+import { patch } from '@n1ru4l/json-patch-plus';
+import { GroupedEventDBScheme, RepetitionDBScheme } from '@hawk.so/types';
+
+/**
+ * One of the features of the events is that their repetition is the difference
+ * between the original, which greatly optimizes storage. So we need to restore
+ * the original repetition payload using the very first event and its difference
+ * between its repetition
+ *
+ * @param originalEvent - the very first event we received
+ * @param repetition - the difference with its repetition, for the repetition we want to display
+ * @returns fully assembled payload of the current repetition
+ */
+export function repetitionAssembler(originalEvent: Record<string, any>, repetition: { [key: string]: any }): any {
```

Comment:
mark this method as deprecated and set removal date

Reply:
done